helm/superset/templates/_helpers.tpl

```diff
@@ -86,11 +86,11 @@ SQLALCHEMY_TRACK_MODIFICATIONS = True
 class CeleryConfig:
   imports  = ("superset.sql_lab", )
   {{- if .Values.supersetNode.connections.redis_password }}
-  broker_url = f"redis://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
-  result_backend = f"redis://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
+  broker_url = f"{env('REDIS_DRIVER')}://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
+  result_backend = f"{env('REDIS_DRIVER')}://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
```

Comment:
Can you include `CACHE_CONFIG` also?
do we need to add `REDIS_DRIVER` to `secrets-env.yaml` also so that the added new default `redis_driver` on values.yaml is set as an env var

Reply:
Thanks for the review! Yes, you are right I should add `REDIS_DRIVER` key and value to `secrets-env.yaml`
CACHE_CONFIG is configured in configOverride section by overriding `CACHE_CONFIG['CACHE_TYPE']` and other values.

For example:

```yaml
configOverrides:
  CACHE_TYPE: "CACHE_CONFIG['CACHE_TYPE'] = 'RedisSentinelCache'"
  CACHE_REDIS_SENTINELS: "CACHE_CONFIG['CACHE_REDIS_SENTINELS'] = [('rfs-redis', 26379)]"
  CACHE_REDIS_SENTINEL_MASTER: "CACHE_CONFIG['CACHE_REDIS_SENTINEL_MASTER'] = 'mymaster'"`
 ```
  At first, I added such changes for `CACHE_CONFIG`, but later I realized that it would be inconvenient and confusing to manage some changes through env and others through `configOverride`, so I deleted those changes. In this case, we need to convert the entire configuration to env variables, which will not be as flexible as using `configOverride`

Reply:
@dpgaspar
Hi! Any updates on this? :)